config_defaults_inc.php

```diff
@@ -4069,6 +4069,13 @@
 ################
 
 /**
+ * Whether sub-projects feature should be enabled.  Before turning this flag OFF,
+ * make sure all sub-projects are moved to top level projects, otherwise
```

Comment:
Move the `Before turning...` bit to the next line for [better phpDoc formatting](http://www.phpdoc.org/docs/latest/guides/docblocks.html#summary) (summary vs description)